packages/utils/src/get-all-content-with-refs.ts

```diff
@@ -0,0 +1,31 @@
+import { BuilderContent, Builder, GetContentOptions } from '@builder.io/sdk';
+import traverse from 'traverse';
+
+async function resolveRefs(content: any, builderInstance: Builder) {
+  const promises: Promise<void>[] = [];
+  const result = traverse(content).map(function (child) {
+    if (child && child['@type'] === '@builder.io/core:Reference') {
+      if (child.model) {
```

Comment:
Should this also maybe check if `child.model && !child.value` to avoid unnecessary fetches if the ref has already been resolved (e.g., from the content API)?